src/client/wasmHelper.ts

```diff
@@ -0,0 +1,36 @@
+declare const atob: any
```

Comment:
I think it is better for move `wasmHelper`  to `util` floder. What do you think ?

Reply:
I think that's good. I was trying to gain DOM types from [src/client/tsconfig.json](), but turns out that doesn't work for code imported during `tsc -p src/node`. 7e0be4e